### PR TITLE
GameState-singleton-object

### PR DESCRIPTION
**Pull request by jparro00**
adding Singleton GameState object to store object references for access from other game objects

### CODE REVIEW

**Review by jparro00 — COMMENTED** [review]
this is a test review comment

File: src/com/spacegame/game/GameState.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+public class GameState {
+
+    private final static Logger log = LogManager.getLogger(GameState.class);
+
+    private GameState(){
+
+    }
+
+    private static GameState instance = null;
```
Comment (TheIrishTrader) [inline src/com/spacegame/game/GameState.java:17]: null

**Review by TheIrishTrader — APPROVED** [review]

**Review by TheIrishTrader — APPROVED** [review]

File: src/com/spacegame/game/GameState.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    public static GameState getInstance() {
```
Comment (jparro00) [inline src/com/spacegame/game/GameState.java:19]: Just a note, this method (i.e., `GameState.getInstance()`)   should be used through out the game to get the current GameState object

File: src/com/spacegame/game/GameState.java (hunk 1 above)
Comment (jparro00) [inline src/com/spacegame/game/GameState.java:17]: I am starting a new conversation to tell you you don't know what your talking about... pfft.. "null" like that means anything

**Review by jparro00 — COMMENTED** [review]
This is my review summary

File: src/com/spacegame/game/GameState.java (hunk 1 above)
Comment (TheIrishTrader) [reply src/com/spacegame/game/GameState.java:17]: That was me practicing writing comments because it was a new feature

Comment (TheIrishTrader) [reply src/com/spacegame/game/GameState.java:17]: thanks for the new conversation.

Comment (jparro00) [reply src/com/spacegame/game/GameState.java:17]: Here is my advice for you in a task list:
- [ ] stop being dumb
- [ ] quit being prick
- [ ] execute `uthuran.fuckYourself(new Dildo("black destroyer"))`

**Pull request merged**